<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Charger Image</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>650</width>
      <height>550</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="imageWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>648</width>
       <height>548</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="contoursButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>50</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Contours</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hystAutoButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>130</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Hysteresis Auto</string>
    </property>
   </widget>
   <widget class="QPushButton" name="affineButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>170</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Affinage</string>
    </property>
   </widget>
   <widget class="QPushButton" name="completeButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>250</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Completion</string>
    </property>
   </widget>
   <widget class="QPushButton" name="seuilGlobalButton">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>50</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Seuil Global</string>
    </property>
   </widget>
   <widget class="QPushButton" name="seuilLocalButton">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>90</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Seuil Local</string>
    </property>
   </widget>
   <widget class="QPushButton" name="baseButton">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Base</string>
    </property>
   </widget>
   <widget class="QComboBox" name="typeFilterBox">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>290</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="chainesButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>210</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Chaines</string>
    </property>
   </widget>
   <widget class="QPushButton" name="penteButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>350</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Pente couleur</string>
    </property>
   </widget>
   <widget class="QPushButton" name="contoursMultiButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>90</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Contours Multi</string>
    </property>
   </widget>
   <widget class="QPushButton" name="colorGradientButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>390</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Gradient couleur</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>470</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>490</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="maxlabel">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>470</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="minlabel">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>490</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hysteSetButton">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>511</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Hysteresis</string>
    </property>
   </widget>
   <widget class="QLabel" name="hystHeadLabel">
    <property name="geometry">
     <rect>
      <x>678</x>
      <y>440</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
